### PR TITLE
LPS-148044 Poshi Automation for Bug LPS-147234 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-148044

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase [hunk 1]
```diff
@@ -1103,6 +1103,22 @@ definition {
 			webContentTitle = "WC WebContent Title");
 	}
 
+	@description = "This is a use case for LPS-147234. View web content with image field after upgrade from DXP 7.1.10."
+	@priority = "3"
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase:1107]: Raise to 4. It's weird ticket doesn't have FP.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase [hunk 2]
```diff
@@ -1103,6 +1103,22 @@ definition {
...
+	test ViewWebContentWithImageArchive7110 {
+		property data.archive.type = "data-archive-web-content-with-image";
+		property database.types = "db2,mariadb,mysql,oracle,postgresql,sqlserver,sybase";
+		property portal.version = "7.1.10";
+
+		task ("View image is shown within web content after upgrade") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase:1113]: View image is shown in field after upgrade

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments?

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments?

Hi @Tim-Cao! PR updated, would you have a look at? Thanks!

**Comment by Tim-Cao** [comment]
LGTM! @ozysouza

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**